pkg/registry/registry.go

```diff
@@ -10,11 +10,12 @@ import (
 type Pair struct {
 	Key       string
 	Value     []byte
+	VersionID string
```

Comment:
add a one-liner above these to explain the purpose of these fields. Maybe add that this internally maps to the `ModifyIndex` field of `consul`.